### DISCUSSION TITLE
How can I build libvips-8.3 release

### DISCUSSION THREAD

**Discussion by muratmirgun** [libvips/libvips#2312] [Q&A] [answered]
I can't make a build because. I also couldn't find the contents of the file when I examined it, I wonder if it was in another internal folder

**Comment by jcupitt** [comment, marked as answer]
Hello @muratmirgun,

Are you certain you want to build libvips 8.3? It's very old.

If you really do need to build it, the source code is on github in the usual place:

https://github.com/libvips/libvips/releases/tag/v8.3.0

**Reply by muratmirgun** [reply]
I need maintain a old project :) maybe I need rewrite that. Thanks for answer.

**Reply by jcupitt** [reply]
Ah gotcha. 

libvips is supposed to be binary compatible back to 8.2 I think, so 8.11 ought to work. There have been some small changes to the way metadata is handled which could cause breaks.
